### PR TITLE
Mvh it3

### PR DESCRIPTION
**Pull request by mikez321**
Create tons of helper methods and tests.  All it3 methods pass spec harness, 1st method of it4 does not work yet.

### CODE REVIEW

File: data/little_game_teams.csv [hunk 1]
```diff
@@ -1,5 +1,9 @@
 game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```
Comment (aperezsantos) [inline data/little_game_teams.csv:1]: Shorter fixture files are definitely easier to work with.

**Review by colintalex — APPROVED** [review]

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -50,12 +50,243 @@ def percentage_ties
...
+    # averages = {}
+    # wins_in_season(season).each do |team_id, wins|
+    #   averages[team_id] = wins / games_by_team_by_season(season)[team_id].to_f
+    # end
+    # averages
+    # require "pry"; binding.pry
+    # head_coaches(season)[averages.key(averages.values.min)]
+  end
+
+  ###### move these methods somewhere else
```
Comment (aperezsantos) [inline lib/stat_tracker.rb:104]: Great work on the helper methods. Utilizing inheritance will help clean up the stat tracker, and abide by SRP.

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -37,22 +39,165 @@ def test_it_can_return_the_highest_total_score
...
+    assert_equal [2012030221, 2012030121, 2012030311, 2012020701, 2012020587], @stat_tracker.game_ids_in_season(20122013)
+  end
+
+  def test_it_knows_the_number_of_games_played_by_teams_in_a_season
+    expected = {3 => 2, 5 => 3, 10 => 1, 26 => 1, 22 => 1}
+    assert_equal expected, @stat_tracker.games_by_team_by_season(20152016)
+  end
+
+  def test_it_knows_coachs_by_season
+    expected_1213 = {
```
Comment (colintalex) [inline test/stat_tracker_test.rb]: Can we change this to expected only? Want to be sure we don't lose track of this

**Review by aperezsantos — APPROVED** [review]

**Review by colintalex — CHANGES_REQUESTED** [review]

File: test/stat_tracker_test.rb (hunk 3 above)
Comment (mikez321) [reply test/stat_tracker_test.rb]: Should be fixed now, I had made some changes but forgot to commit them before pushing this branch.  Good catch!

**Review by colintalex — APPROVED** [review]
Looks good!

**Pull request merged**